RecoPixelVertexing/PixelLowPtUtilities/plugins/PixelClusterShapeExtractor.cc

```diff
@@ -0,0 +1,407 @@
+// VI January 2012: needs to be migrated to use cluster directly
```

Comment:
please move this to /test to remove dependency on Sim for the lib used in production